components/Comment.js

```diff
@@ -26,14 +31,25 @@ class Comment extends React.Component {
       modified: false,
       comment: props.comment,
       mode: undefined,
+      showDeleteModal: false,
     };
 
     this.save = this.save.bind(this);
     this.handleChange = this.handleChange.bind(this);
     this.toggleEdit = this.toggleEdit.bind(this);
     this.messages = defineMessages({
-      edit: { id: 'comment.edit', defaultMessage: 'edit' },
-      cancelEdit: { id: 'comment.cancelEdit', defaultMessage: 'cancel edit' },
+      edit: { id: 'comment.edit', defaultMessage: 'Edit' },
+      cancelEdit: { id: 'comment.cancelEdit', defaultMessage: 'Cancel edit' },
+      delete: { id: 'comment.delete', defaultMessage: 'Delete' },
+      'delete.modal.cancel': { id: 'delete.modal.cancel', defaultMessage: 'Cancel' },
```

Comment:
The generic `Cancel` should be usable here, no need to create a new one

```suggestion
      'delete.modal.cancel': { id: 'cancel', defaultMessage: 'Cancel' },
```